apps/OpenSpace/ext/launcher/include/sgctedit/settingswidget.h

```diff
@@ -72,17 +74,81 @@ Q_OBJECT
      */
     void setShowUiOnFirstWindow(bool setUiOnFirstWindow);
 
+    /**
+     * Sets value for whether or not the checkbox for having the UI only on the first
+     * window is enabled. This checkbox should only be clickable if the number of
+     * windows is 2 or more. 
+     */
+    void setEnableShowUiOnFirstWindowCheckbox(bool enable);
+
+    /**
+     * Gets the value of the selection for which display first window should mirror if
+     * the option to show the Ui in the first window is enabled. Note that this will
+     * return a value even if the checkbox is not enabled.
+     * 
+     * \return -1 if in a disabled state (e.g. when showUiOnFirstWindow() returns false)
+     *          0 if no window is being mirrored (only the UI will appear in first
+     *            window, with no 3D rendering)
+     *          1 if the second window's viewport is mirrored in the first window
+     *          2 if the third window's viewport is mirrored in the first window
+     *          3 if the fourth window's viewport is mirrored in the first window
+     */
+    int graphicsSelectionForShowUiOnFirstWindow() const;
```

Comment:
I think it might be easier to use if this function would return an `enum class` that has a value for each of the numeric return values

Reply:
I had forgotten to change the comment header which was outdated. I think it's less confusing now because the value matches the 1-based window number to use. If set to zero then no window is used (no graphics).
I prefer leaving it as an integer since that is what QComboBox uses, and the values for this and its set function are only used with that object.